Dockerfile

```diff
@@ -39,4 +39,4 @@ USER node
 EXPOSE 8443
 
 # Run the application.
-CMD node build/src/server.js
+CMD cd build && node src/server.js
```

Comment:
The magic that took hours to figure out. Change the `pwd` to ensure we never risk reading source files again, ez.